README.md

```diff
@@ -29,15 +29,15 @@
 
 # JDA (Java Discord API)
 
-JDA strives to provide a clean and full wrapping of the Discord REST api and its Websocket-Events for Java.
-This library is a helpful tool that provides the functionality to create a discord bot in java.
+JDA strives to provide a clean and full wrapping of the Discord REST API and its websocket gateway events for the JVM.
```

Comment:
I get the idea here but "for Java" makes more sense here (and is less confusing considering it's "_Java_ Discord API" and not "JVM Discord API").

Reply:
> I get the idea here but "for Java" makes more sense here (and is less confusing considering it's "Java Discord API" and not "JVM Discord API").

Well seeing as it's a Java "centered" library the word Java can be replaced with JVM because addons like JDA-KTX are supported. The library is still referenced to as JDA, though explicitly stating that it supports the JVM as a whole indicates that it's available for all languages.

Reply:
I would have to agree with Chew on this one, most of the time people say Java instead of JVM for libraries that are primarily used in Java opposed to Kotlin. The optional ability to write a bot in Kotlin (or to use Kotlin libraries) is implied. The rest of the changes look good though.